run.sh

```
@@ -20,11 +24,25 @@ set -a
+export LOG_FORMAT=console          # human-readable colored logs
+# export CREWAI_VERBOSE=true         # show agent reasoning in console
+
 # Support both APP_PORT (new) and PORT (legacy) variables with a sane default
 APP_PORT="${APP_PORT:-${PORT:-5000}}"
 export APP_PORT
 export PORT="$APP_PORT"
 
+# When running locally (not inside Docker Compose), HOST_ROBOT_TESTS_DIR must be
+# an absolute Windows path so the Docker daemon can resolve the bind mount.
+# The .env value (./robot_tests) is relative and only valid inside Docker Compose.
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd -W 2>/dev/null || pwd)"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)